typescript/lib/services/service_context.ts

```diff
@@ -65,25 +65,34 @@ class ServiceContext {
     }
 
     // Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
-    public execCommand(command: string[]): [number, Uint8Array | string, Error] {
+    public async execCommand(command: string[]): Promise<[number, Uint8Array | string, Error]> { //TODO - need to bubble up async, how would users know to make functions that depend on this async?
```

Comment:
basically, because they'll have no choice - either they handle the promise synchronously themselves (e.g. a `while promise.isNotResolved()` loop), or they decide to use `await` themselves which means tehy need to make their parent function `async`